101-synapse-poc/scripts/getobjectid.ps1

```diff
@@ -1,16 +1,7 @@
-try {
-    Import-Module AzureAD
-    Write-Host "Azure Active Directory Module exists"
-} 
-catch {
-    Write-Host "Module does not exist - installing Azure Active Directory Module"
-    Install-Module AzureAD -AllowClobber
-    Import-Module AzureAD
-}
+Connect-AzAccount
+$userUPN = Read-Host -Prompt 'Enter your User Principal Name'
```

Comment:
You can get the upn from the context:

(Get-AzAdUser -UserPrincipalName (Get-AzContext).account.id).id